### PR TITLE
bump min version to 13 for ios project

### PR DESCRIPTION
**Pull request by waterskier2007**
### Issue Link :link:

Attempts to fix testing issues here https://github.com/danielgindi/Charts/pull/5028

### Goals :soccer:
 
Fix testing issues for release 5.0.0

### Implementation Details :construction:

Just bumps the iOS demo project min version to 13 (and tvOS)

### Testing Details :mag:

This should fix tests as reported here https://github.com/danielgindi/Charts/pull/5028#issuecomment-1537166577

### CODE REVIEW

File: Charts.xcodeproj/project.pbxproj
Comment (FelixHerrmann) [inline Charts.xcodeproj/project.pbxproj:1]: Hey, not sure if this was intended, but there is no need to bump the frameworks minimum deployment target to 13

Comment (waterskier2007) [reply Charts.xcodeproj/project.pbxproj:1]: good point, updated

**Review by FelixHerrmann — APPROVED** [review]

File: Charts.xcodeproj/project.pbxproj
Comment (FelixHerrmann) [reply Charts.xcodeproj/project.pbxproj:1]: Thx!

**Comment by waterskier2007** [comment]
@pmairoldi or @danielgindi thoughts on getting this merged in to try and resolve the test issues on the release branch?

**Comment by pmairoldi** [comment]
Thank you! Sorry I've been pretty busy.

**Pull request merged**